superset-frontend/src/dashboard/components/Header.jsx

```diff
@@ -86,6 +89,7 @@ const propTypes = {
   setMaxUndoHistoryExceeded: PropTypes.func.isRequired,
   maxUndoHistoryToast: PropTypes.func.isRequired,
   refreshFrequency: PropTypes.number.isRequired,
+  isPersistentRefreshFrequency: PropTypes.number.isRequired,
```

Comment:
this variable is a number but starts with `is` which would imply it's a boolean. Can we rename the variable?

Reply:
actually, i see this same variable set to booleans further down, is the proptypes wrong here?

Reply:
correct. this should be boolean type